src/README.md

```diff
@@ -0,0 +1,30 @@
+# Command line apps in Rust
+
+Rust is a statically compiled, fast language with great tooling and a rapidly growing ecosystem.
+That makes it a great fit for writing command line applications:
+They should be small, portable, and quick to run.
```

Comment:
I think most people care less about how long their code actually runs, but instead that there's no startup cost.

Reply:
That's what I meant with "quick to run". I'll see if I can improve that wording a bit more.